.doc_gen/metadata/services.yaml

```diff
@@ -80,6 +105,16 @@ eventbridge:
   short: "&EV;"
   sort: "EventBridge"
   version: eventbridge-2015-10-07
+iam:
+  long: "&IAMlong;"
+  short: "&IAM;"
+  sort: "IAM"
+  version: iam-2010-05-08
```

Comment:
IAM is already in this file (sorted at the end, for some reason).

Reply:
I'll remove this block.

Reply:
Removed.